service-catalog/gcp-backup/README.md

```diff
@@ -0,0 +1,132 @@
+# GCP backup
```

Comment:
I think GCP is an implementation detail and would therefore rename this piece of our infrastructure (i.e. the document). Internally, we've talked about this in terms of "backup of critical (Rust) assets". How about we use a similar name for this "service"?